client/src/API/Ratings.js

```diff
@@ -1,16 +1,36 @@
-import GITHUB_ACCESS_TOKEN from '../auth.js';
-
+const fetch = require('node-fetch');
 const Ratings = {
-  getReviewMetadata: (id) => {
-    return fetch(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/reviews/meta?product_id=${id}`, {
+  getReviewList: (product_id, sort, page, count) => {
+    return fetch('http://127.0.0.1:3000/reviews/?' +
```

Comment:
return fetch('http://localhost:3000/reviews/?'

Reply:
Sweet thank you for figuring this out about using localhost:3000 instead of the ip address. For some reason I wasnt able to get it to work, but now its working just fine!